lua/telescope/actions/init.lua

```diff
@@ -272,6 +272,28 @@ function actions.close_pum(_)
   end
 end
 
+--- Copy current entry command to a specific register or unnamed plus as default
+--- Don't let user use  blackhole register (_), let them choose instead
+---@param prompt_bufnr number: The prompt bufnr
+---@param opts table: options to pass to the action
+---@field reg: the register which the content will be sent
+function actions.copy_command_to_reg(prompt_bufnr, opts)
+  local validate_reg = function(reg)
+    while reg == '_' do
```

Comment:
Why not the simple solution? I don't see why this would be more careful? Users have to confirm with `<CR>` on `vim.fn.input.

Reply:
Ah, I thought about someone who kept trying to use '_'. I went too far, sorry.

Reply:
If you now add docs for `_`, I'll give it a whirl later today or tomorrow and then I think we can merge :)

Reply:
I think everything is done (sorry, I forgot to send a message).